src/content-script.ts

```diff
@@ -74,17 +75,39 @@ async function main() {
 		return;
 	}
 
-	const onOpenSavedRepliesButtonClick = () => {
+	async function getSoon<Value extends Node | NodeList | null>(
+		getter: () => Value,
+	) {
+		for (let i = 0; i < 100; i += 1) {
+			const result = getter();
+			if (result && (!(result instanceof NodeList) || result.length)) {
+				return result;
+			}
+
+			await new Promise((resolve) => setTimeout(resolve, i));
+		}
+
+		// TODO: Add handling for a rejection
+		// https://github.com/JoshuaKGoldberg/refined-saved-replies/issues/2
+		throw new Error(`Element was never found 😔`);
+	}
+
+	const onOpenSavedRepliesButtonClick = async () => {
 		// 7. Add the new replies to the saved reply dropdown
-		const replyCategoriesDetailsMenus = document.querySelectorAll(
-			`markdown-toolbar details-menu[src^="/settings/replies?context="]`,
+		const replyCategoriesDetailsMenus = await getSoon(() =>
+			document.querySelectorAll(
+				`div.Overlay-body fuzzy-list focus-group div[data-view-component]`,
```

Comment:
A more stable selector here would be `.Overlay-body .js-saved-reply-menu`

Reply:
Dangit, merged and _then_ saw these comments. Thanks! Filing a followup issue...

Reply:
#168